code/data_structures/avl_tree/readme.md

```diff
@@ -0,0 +1,26 @@
+# AVL Tree
+Description
+---------
+It is a **self balancing** binary search tree, where the difference between heights of left and right subtrees cannot be more than one for all nodes. In other words, the **balance factor** for each node in an AVL tree cannot exceed 1.
+Where, > balance factor = height of left subtree - height of right subtree
```

Comment:
I thought the balance factor was right subtree - left subtree, no? See [wiki](https://en.wikipedia.org/wiki/AVL_tree#Balance_factor) for reference.